Content.Client/GameObjects/Components/Chemistry/ChemMaster/ChemMasterBoundUserInterface.cs

```diff
@@ -0,0 +1,102 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using Content.Client.GameObjects.Components.Chemistry.ChemMaster;
+using Content.Shared.Chemistry;
+using Content.Shared.GameObjects.Components.Chemistry;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects.Components.UserInterface;
+using Robust.Client.UserInterface;
+using Robust.Client.UserInterface.Controls;
+using Robust.Shared.GameObjects.Components.UserInterface;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+using Robust.Shared.Maths;
+using static Content.Shared.GameObjects.Components.Chemistry.SharedChemMasterComponent;
+
+namespace Content.Client.GameObjects.Components.Chemistry
+{
+    /// <summary>
+    /// Initializes a <see cref="ChemMasterWindow"/> and updates it when new server messages are received.
+    /// </summary>
+    [UsedImplicitly]
+    public class ChemMasterBoundUserInterface : BoundUserInterface
+    {
+#pragma warning disable 649
+        [Dependency] private readonly ILocalizationManager _localizationManager = default!;
```

Comment:
You don't need to pull in `ILocalizationManager` as a dependency. You can use the `Loc` static instead.